app/initializers/tracer.js

```diff
@@ -26,7 +26,7 @@ class Tracer {
     this.installXHR();
   }
 
-  // xhr interceptor based on the symfony profiler
+  // xhr interceptor based on the Symfony profiler
```

Comment:
This is a proper name, capitalizing it tells other spellcheckers it's intentional (and hints to normal readers that it's the name of the profiler...)